src/poetry/vcs/git/backend.py

```diff
@@ -331,16 +332,24 @@ def _clone_submodules(cls, repo: Repo) -> None:
         repo_root = Path(repo.path)
         modules_config = repo_root.joinpath(".gitmodules")
 
+        # A relative URL by definition starts with ../ or ./
+        relative_submodule_regex = re.compile(r"^(\.{1,2})/?")
```

Comment:
why are we capturing the dots?  what is the question mark for?

To reiterate: ny previous suggestion was `r"^\.{1,2}/"`

Reply:
Sorry, misread it. Done.